test-infra/auto-deploy/deploy-cron-master.yaml

```diff
@@ -14,14 +14,18 @@ spec:
         spec:
           containers:
           - name: deploy-worker
-            image: gcr.io/kubeflow-ci/deploy-worker:v20190302-afc8ef8-dirty-201fe5
+            image: gcr.io/kubeflow-ci/deploy-worker:v20190403-33a1352-e3b0c4
             env:
             - name: GOOGLE_APPLICATION_CREDENTIALS
               value: /secret/gcp-credentials/key.json
+            - name: CLIENT_ID
```

Comment:
Can we load CLIENT_ID and CLIENT_SECRET from an OAuth secret as opposed to hard  coding the values?

How are we doing this right now? Aren't we loading the keys from a GCS file?

Reply:
done.